src/e

```diff
@@ -155,7 +155,14 @@ program
 program
   .command('update-goma')
   .description('Ensure a fresh copy of Goma is installed')
-  .action(() => console.log(`goma checksum: ${goma.downloadAndPrepare()}`));
+  .action(() => {
+    const args = process.argv.slice(3);
+    const config = {};
+    if (args.length > 0 && args[0] === 'msftGoma') {
```

Comment:
Why can't we load the actual config here?

Reply:
@MarshallOfSound I wanted a way to download Goma without having to setup a config